### PR TITLE
Fix Nightly 7/20

### PR DESCRIPTION
**Pull request by armstrongnate**
[ignore-commit-lint]
[run-nightly]

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/hj1jr9h) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fhj1jr9h)
teacher: [Link](https://rebrand.ly/uu0mf2t) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fuu0mf2t)
parent: [Link](https://rebrand.ly/w6cbjds) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fw6cbjds)

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.33% | 93.33% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by armstrongnate** [comment]
@cobbal I added it here. Thanks.

File: Student/StudentE2ETests/Planner/PlannerTests.swift [hunk 1]
```diff
@@ -56,7 +56,7 @@ class PlannerTests: CoreUITestCase {
 
         PlannerList.event(id: "2233").tap()
         app.find(label: "first").waitToExist()
-        XCTAssert(app.find(label: "Instructure SLC").exists())
+        app.find(label: "Instructure SLC").waitToExist()
```
Comment (cobbal) [inline Student/StudentE2ETests/Planner/PlannerTests.swift:59]: Could you make this change in the Parent version of PlannerTests too?

Comment (armstrongnate) [reply Student/StudentE2ETests/Planner/PlannerTests.swift:59]: Oh right, thanks!

**Pull request merged**